types/gl-vec3/dist.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Calculates the euclidian distance between two number's. Aliased as dist.
+ */
+declare function dist(a: number[], b: number[]): number[];
```

Comment:
Should this just return number?
https://github.com/stackgl/gl-vec3/blob/219dbe7bd23426fb921ee8f53edeb33e7a308155/distance.js#L14

Reply:
@jessetrinity Good pickup! I just noticed that I made this mistake in a couple of places (`dist`, `angle`, `len` etc ).

 I've updated the branch, ready for another review.